Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -138,6 +138,10 @@ internal static int IdFromTitle(string title, Type type)
 
 		static int GetId(Type type, string memberName)
 		{
+			// This may legitimately be null in designer scenarios
+			if (type == null)
```

Comment:
The behavior of the previewer when the state of the project is not compiled is to not display any images? Users will still need to compile that app in order for drawables to show up? Just validating the expectations of the viewer when app is in a non compiled state

Reply:
At this stage, yes, we just want to fix the NRE as it will break many common scenarios, and it is too close to shipping Dev 16 to do more.

Post 16.0 , however, we'd like to investigate options to actually display the images.

Reply:
Yeah - the expectation is that images should (and will) just work. That is a separate piece of work, as Tim said. We're not even looking at it currently :)